cmds/vpdbootmanager/addentry/netboot.go

```diff
@@ -0,0 +1,56 @@
+package addentry
```

Comment:
this is small enough that it could be part of the vpdbootmanager package instead of its own